testing/scalar-functions.test

```diff
@@ -243,3 +243,26 @@ do_execsql_test date-with-invalid-timezone {
   SELECT date('2023-05-18 15:30:45+25:00');
 } {{}}
 
+do_execsql_test length-double-quoted-text-non-column {
+  SELECT length("limbo") from users LIMIT 1;
```

Comment:
These should not work because `"limbo"` is not a column right? (i.e. single quotes work for arbitrary strings, double quotes are for schema/table/column references)